src/test/java/edu/ucla/library/services/util/props/PropsWriterTest.java

```diff
@@ -0,0 +1,132 @@
+
+package edu.ucla.library.services.util.props;
+
+import static com.github.stefanbirkner.systemlambda.SystemLambda.catchSystemExit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.junit.contrib.java.lang.system.SystemErrRule;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+
+/*
+ * Unit tests for ProsWriter.java
+ */
+public class PropsWriterTest {
+
+  /**
+   * Default properties file.
+  */
+    private static final String DEFAULTS =
+        "src/test/resources/properties.default";
+
+  /**
+   * Output template file.
+  */
+    private static final String TEMPLATE =
+        "src/test/resources/properties.tmpl";
+
+  /**
+   * Final output file.
+  */
+    private static final String OUTPUT =
```

Comment:
Since this is an output file that's generated by the test, I think it should be put in System's tmp dir or in the `target` directory. We have the cleanup in the test cleanup method, but if the test gets killed before getting to that part it would be good to have it in a place that's periodically cleaned.

Actually, I think since we want more than one happy path test, it would be best to generate a name for the file (using UUID.randomUUID().toString()) or something. That way the output file is specific to a particular test (and has no chance of being reused). You can set that in the Before setup and clean it up in the After cleanup again.